.github/workflows/release.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         with:
           python-version: "3.12"
       - name: "Install Python Packages"
-        run: "pip install poetry"
+        run: "pip install poetry==2.0.0"
```

Comment:
@HanlinMiao To have successful releases, #6725 and #6726 might need to add pinning of poetry to the appropriate 1.x versions in their respective `release.yml` workflows?